entity-service/src/main/kotlin/com/egm/stellio/entity/service/EntityOperationService.kt

```diff
@@ -279,7 +282,12 @@ class EntityOperationService(
                     )
 
                     creations.add(entity.id)
-                    entityService.publishCreationEvent(entity)
+                    entityService.getSerializedEntityById(entity.id)?.also {
```

Comment:
We have to be consistent, events have to be triggered from the same logical place (web handlers in this case). 

Code will be harder to read and maintain if events are sometimes sent to web handlers, and other times from the services.